### DISCUSSION TITLE
Do you like running rm -rf --no-preserve-root / just for fun on v86?

### DISCUSSION THREAD

**Discussion by hello-smile6** [copy/v86#498] [Show and tell]
I do. Go to https://copy.sh/v86/?profile=archlinux&c=rm%20-rf%20--no-preserve-root%20/ and watch Arch Linux self-destruct automatically! @copy

**Comment by raspiduino** [comment]
It just said `rm: cannot remove: Operation not permitted!`

**Reply by hello-smile6** [reply]
Strange. I hope that isn't on a real machine!

**Reply by abundant-ray** [reply]
i have done it on a fresh linux install. i dont think my computer liked it :|

**Reply by hello-smile6** [reply]
> i have done it on a fresh linux install. i dont think my computer liked it :|

I don't think so either.

**Reply by 1jammer1** [reply]
> > i have done it on a fresh linux install. i dont think my computer liked it :|
> 
> I don't think so either.

yeah me I did that too now my computer go panic
